src/components/contact-form.svelte

```diff
@@ -74,7 +75,39 @@
     input: string;
   }>();
 
-  const budgetOptions = ['10k - 25k', '25k - 50k', '50k - 100k', '100k+'];
+  const budgetOptions = [
+    {
+      budget: '15.000€ to 50.000€',
+      description:
+        'For small or pilot projects, like marketing sites, PoCs, or branding, like  <a class="text-foreground-accent underline" href="/projects/yonder">Kota</a>. Ranging from 1 to 3 months.'
+    },
+    {
+      budget: '50.000€ to 100.000€',
+      description:
+        'For mid-size projects, like corporate websites or MVPs, like <a class="text-foreground-accent underline" href="/projects/passaporte-natura">PN2000</a>. Ranging from 3 to 6 months.'
+    },
+    {
+      budget: '100.000€ to 200.000€',
+      description:
+        'For projects like tailor-made e-commerces and mid-sized products, like <a class="text-foreground-accent underline" href="/projects/dia">Dia</a>. Ranging from 6 to 9 months.'
+    },
+    {
+      budget: '200.000€ to 300.000€',
+      description:
+        'For full-fledged, single-platform products of a higher complexity, like <a class="text-foreground-accent underline" href="/projects/bion">Bion</a>. Ranging from 6 to 12 months.'
+    },
+    {
+      budget: '300.000€ to 400.000€',
+      description:
+        'For full-fledged, multi-platform, larger products like <a class="text-foreground-accent underline" href="/projects/allo">allO</a>. Around a 1 year project.'
+    },
+    {
+      budget: '400.000€ and above',
+      description:
+        'For full-fledged, multi-platform, largely-complex products, like <a class="text-foreground-accent underline" href="/projects/oopsie">Oopsie</a>. Around a 1 year project.'
+    }
```

Comment:
We aren't making these translatable as we are doing with every other text in the form